gradle.properties

```
@@ -12,7 +12,7 @@ ballerinaGradlePluginVersion=1.0.0
 gsonVersion=2.8.8
 
 stdlibIoVersion=1.3.1
-stdlibHttpVersion=2.5.2
+stdlibHttpVersion=2.5.4-20231013-152300-784fd78
```

Need to update http version after release
```suggestion
stdlibHttpVersion=2.5.4
```